docs/platforms-release-process.md

```diff
@@ -122,6 +123,10 @@ Double check you replace "Android" in the subject and mail body - there is no un
 
 Note that it would be possible to continue with some of the [Before Release](#before-release) items while waiting for a possible response.
 
+### Ensure changes are in a dev version
+
+The procedure described here may leave non-master branch with a non-dev version number. Functional changes should always be done in a "-dev" version. Please mark the "-dev" version manually or using cordova-coho before making changes, and test it especially when marking "-dev" version manually.
```

Comment:
What is the `cordova-coho` command to use?
Where does one have to change things when doing this manually?

Reply:
(Sorry about being pedantic here, but I guess you probably have this information handy)